packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -743,6 +781,14 @@ export class Function extends FunctionBase {
       securityGroups = [securityGroup];
     }
 
+    if (props.filesystems) {
+      for (const fs of props.filesystems) {
+        if (fs.config.securityGroups) {
+          securityGroups.push(...fs.config.securityGroups);
```

Comment:
Isn't this only going to work if the file system security group has an inbound rule allowing traffic from its own security group (`allowInternally` in terms of CDK)?

Reply:
can we verify/mutate the security group?

Reply:
OK we don't need security group anymore. :-)